cli/medperf/entities/benchmark.py

```diff
@@ -58,10 +58,12 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
 
-        self.generated_uid = f"p{self.data_preparation_mlcube}m{self.reference_model_mlcube}e{self.data_evaluator_mlcube}"
+    @property
+    def local_id(self):
+        return self.name
```

Comment:
AFAIU now generated_uid is replaced with local_id. But where is this field `name` taken from? I don't see how it is built now...

Reply:
now the local ID is changed from `p{self.data_preparation_mlcube}m{self.reference_model_mlcube}e{self.data_evaluator_mlcube}` to just the name of the benchmark. That's it